packages/nx/src/utils/package-manager.spec.ts

```diff
@@ -58,6 +62,27 @@ describe('package-manager', () => {
       expect(fs.existsSync).toHaveBeenCalledTimes(3);
     });
 
+    it('should detect bun package manager from bun.lockb', () => {
+      jest.spyOn(configModule, 'readNxJson').mockReturnValueOnce({});
+      jest.spyOn(fs, 'existsSync').mockImplementation((p) => {
+        switch (p) {
+          case 'yarn.lock':
+            return false;
+          case 'pnpm-lock.yaml':
+            return false;
+          case 'package-lock.json':
+            return false;
+          case 'bun.lockb':
+            return true;
+          default:
+            return jest.requireActual('fs').existsSync(p);
+        }
+      });
+      const packageManager = detectPackageManager();
+      expect(packageManager).toEqual('bun');
+      expect(fs.existsSync).toHaveBeenCalledTimes(3);
```

Comment:
Can someone explain to me how this could possible be 6 like NPM 1 :/ The two failing tests are not making sense